[Blog] GSoC new phase announcement (#5989)
jenkins-infra/jenkins.io: content/blog/2023/02/2023-02-01-gsoc-update.adoc

Although we are waiting for Google's decision, we will start the next GSoC phase without delay.
In this phase, the candidates will prepare their detailed GSoC application with the help of the Jenkins Community.
The proposals are based on the link:/projects/gsoc/2023/project-ideas/[2023 project ideas].
The phase will culminate with the selection of the students/contributors participating in in this year's GSoC. 
The deadline for the application is April 4.

Read carefully the guidelines for a successful application on the link:/projects/gsoc/students/[Information for GSoC Contributors] page.

In a nutshell, select a project idea and begin thinking on how you will build a proposal to convince the mentors that you are the best candidate to bring it to a successful end.

We will help you through the currently available channels (Community forum and Gitter) but also with weekly on-line link:https://en.wiktionary.org/wiki/office_hours[office-hours].
During these meetings you can ask for any clarification, disambiguation, precision on the project ideas or about the GSoC program.
These sessions will be recorded so that everyone can benefit from them.

> Before setting up the weekly Office Hour, we will organize a poll to choose a timing that suits the timezones of most of the candidates.
> Last year we held sessions at 14h00 UTC for Europe and Asia and 03h00 UTC for the US and the Pacific.

Reviewer: ```suggestion
Before setting up the weekly Office Hours, we will organize a poll to determine a time that suits the timezones of most of the candidates.
Last year, we held sessions at 14h00 UTC for Europe and Asia, and 03h00 UTC for the US and the Pacific.
```
small suggestion for formatting/syntax

> When the Jenkins participation to GSoC 2023 is confirmed you can actively start working on your draft proposal.
> Make these drafts available to the Jenkins Community for review via https://community.jenkins.io.
> The previous year's experience shows that proposals that didn't use this public discussion and review process turn out to be of to poor quality to be selected.

Reviewer: ```suggestion
When the Jenkins participation in GSoC 2023 is confirmed, you can actively start working on your draft proposal.
Make these drafts available to the Jenkins Community for review via link:https://community.jenkins.io[the community forum].
Previous years' experience shows that proposals not submitted for the public discussion and review process, are not selected due to being less convincing.
```
small suggestion for wording, just trying to word this idea so that it is clear the discussion and review are critical to success

> It is probably a new and uncomfortable way of approaching such candidatures for you.
> But it is the Open Source way, it is rewarding, and will give you the best chances to be selected.     

Reviewer: ```suggestion
This may be a new and uncomfortable way of approaching such candidatures for you, but it is the Open Source way Further discussion and review is rewarding and gives you the best chances to be selected.     
```
small wording suggestion for readability

> Use as much help as you can: it is the best way to have a strong and successful proposal. 
> Actively use the link:https://community.jenkins.io/c/contributing/gsoc/6[GSoC Community topic], the link:https://gitter.im/jenkinsci/gsoc-sig[GSoC Gitter channel], or the Office Hour sessions.

Reviewer: ```suggestion
Collaborate as much as you can, as it is the best way to have a strong and successful proposal. 
Actively use the link:https://community.jenkins.io/c/contributing/gsoc/6[GSoC Community topic], the link:https://gitter.im/jenkinsci/gsoc-sig[GSoC Gitter channel], or the Office Hour sessions.
```
small suggestion

=== Some useful preparation notes

* link:/blog/2022/11/23/get-prepared-for-gsoc/[GSoC 2023: how to get prepared ]

> * recording of the link:https://youtu.be/k_sTkGtTix8[GSoC 2023: A Guide to Better Preparations for Candidates] session.

Reviewer: ```suggestion
* Recording of the link:https://youtu.be/k_sTkGtTix8[GSoC 2023: A Guide to Better Preparations for Candidates] session.
```
formatting suggestion
